umd-fcrepo-event-router/src/main/java/edu/umd/lib/fcrepo/camel/routing/Router.java

```diff
@@ -60,9 +61,17 @@ public void configure() throws Exception {
         from("direct:batch.queue")
             .log(LoggingLevel.DEBUG, logger, "Routed to batch queue.")
             .choice()
-                .when(and(simple("{{batch.skip.pcdm.container}} == 'true'"),
-                        simple(headerString(JMS_IDENTIFIER) + " == '/pcdm'")))
-                    .log(LoggingLevel.DEBUG, logger, "Suppressing '/pcdm' node event for batch user.")
+                .when(
+                    or(
+                        and(
+                            simple("{{batch.skip.pcdm.container}} == 'true'"),
+                            simple(headerString(JMS_IDENTIFIER) + " == '/pcdm'")
+                        ),
+                        simple(headerString(JMS_IDENTIFIER) + " in '{{batch.skip.paths}}'")
+                    )
+                    )
```

Comment:
If `batch.skip.paths` alone is sufficient to skip `/pcdm`, Can we replace
```
                .when(
                     or(
                         and(
                             simple("{{batch.skip.pcdm.container}} == 'true'"),
                             simple(headerString(JMS_IDENTIFIER) + " == '/pcdm'")
                         ),
                         simple(headerString(JMS_IDENTIFIER) + " in '{{batch.skip.paths}}'")
                     )
                     )
```
with
```
                .when(simple(headerString(JMS_IDENTIFIER) + " in '{{batch.skip.paths}}'"))
```